pkg/controller/upload_controller.go

```diff
@@ -403,10 +403,7 @@ func (c *uploadController) processUpload(req *pluginv1api.Upload) error {
 		return errors.WithStack(err)
 	}
 
-	log.WithFields(logrus.Fields{
-		"phase":      req.Status.Phase,
-		"generation": req.Generation,
-	}).Info("Upload Completed")
+	log.Info("Upload Completed")
```

Comment:
Any reason why you removed Phase and Generation from logs?

Reply:
L278 `log := loggerForUpload(c.logger, req)`, already creates a logger with the fields, no need to specify it again.